alphagov/paas-product-page
# Refresh product pages

What
----

Based on input from the product managers and tech arch we have extended to product documentation to incorporate material that had only been available to prospective tenants in the primer document.

changes as follows:

- restructured `features` page
- new `developer` page
- new `observability` page
- new `constraints` page

How to review
-------------

- pull the branch 
- run the app locally 
- ensure essential wiring of pages works
- review content for inaccuracies  and plain english 

Who can review
--------------
- a product manager
- someone from the technical team
